ranger/core/main.py

```diff
@@ -119,6 +119,9 @@ def main(
         FileManagerAware.fm_set(fm)
         load_settings(fm, args.clean)
 
+        if args.choosedir:
+            SettingsAware.settings.global_inode_type_filter = 'd'
```

Comment:
Either `ranger.container.directory` or some new module like `ranger.core.constants`. Again, I'd like to hear @hut's opinion on this.